pkg/virt-handler/cache/cache_test.go

```diff
@@ -251,6 +256,8 @@ var _ = Describe("Domain informer", func() {
 
 			domain := api.NewMinimalDomain("test")
 			list = append(list, domain)
+			err := ghostRecordStore.Add("default", "test", socketPath, podUID)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
nit: simply style suggestion and from a personal preference:
```golang
Expect(ghostRecordStore.Add("default", "test", socketPath, podUID)).To(Succeeded())
```

Reply:
@fossedihelm Done.